### DISCUSSION TITLE
Full Name

### DISCUSSION THREAD

**Discussion by mcvoigt** [drakkan/sftpgo#300] [Ideas]
Hello.
In addition to the username, it is possible to have the full name in the user's registration.
Ex
username = drakkan
realname, completename or fullname = Nicola Murino

Thanks,
Michael Voigt

**Comment by drakkan** [comment]
Hi,

I'll evaluate this for future updates, meantime you can use the free text field for additional info, thank you

**Comment by mcvoigt** [comment]
Hi,
OK thank you.

Regards, Michael Voigt

Em sáb., 6 de fev. de 2021 às 12:17, Nicola Murino <notifications@github.com>
escreveu:
